financial-connections/src/main/java/com/stripe/android/financialconnections/domain/SaveAccountToLink.kt

```diff
@@ -117,23 +120,27 @@ internal class SaveAccountToLink @Inject constructor(
         manifest: FinancialConnectionsSessionManifest,
         selectedAccounts: Int,
     ) {
+        // No saved accounts means a manually entered account was already attached.
+        val accountsCount = if (selectedAccounts == 0) 1 else selectedAccounts
```

Comment:
Can we remove `<item quantity="zero">…</item>` then?

Also, we can turn this into `val accountsCount = max(1, selectedAccounts)` instead.